ubuntu-20.04-amd64.json

```diff
@@ -341,15 +282,15 @@
     "https_proxy": "{{env `https_proxy`}}",
     "hyperv_generation": "2",
     "hyperv_switch": "{{env `hyperv_switch`}}",
-    "iso_checksum": "f11bda2f2caed8f420802b59f382c25160b114ccc665dbac9c5046e7fceaced2",
+    "iso_checksum": "28ccdb56450e643bad03bb7bcf7507ce3d8d90e8bf09e38f6bd9ac298a98eaad",
     "iso_checksum_type": "sha256",
-    "iso_name": "ubuntu-20.04.1-legacy-server-amd64.iso",
+    "iso_name": "focal-live-server-amd64.iso",
     "memory": "1024",
     "mirror": "https://cdimage.ubuntu.com",
-    "mirror_directory": "ubuntu-legacy-server/releases/20.04.1/release",
+    "mirror_directory": "ubuntu-server/focal/daily-live/current",
```

Comment:
:+1:. Fixed.

Reply:
:+1: fixed by using a specific dated build. Annoyingly Ubuntu Server is only available as daily builds now but they do keep the last three builds around so this will at least make it obvious why it isn't working when the specified build falls off the queue.